src/server/dragonfly_test.cc

```diff
@@ -91,6 +92,28 @@ TEST_F(DflyEngineTest, Sds) {
   sdsfreesplitres(argv, argc);
 }
 
+class DflyRenameCommandTest : public DflyEngineTest {
+ protected:
+  DflyRenameCommandTest() : DflyEngineTest() {
+    // remame flushall to myflushall, flushdb command will not be able to execute
+    absl::SetFlag(&FLAGS_rename_command, "flushall=myflushall,flushdb=,");
```

Comment:
Can you remove the last `,` to make sure that the user doesn't have to input that?

Reply:
ok